llvm/test/CodeGen/AArch64/speculation-hardening.mir

```diff
@@ -11,34 +12,148 @@
 # - other direct branches don't seem to be generated by the AArch64 codegen
 --- |
   define void @nobranch_fallthrough(i32 %a, i32 %b) speculative_load_hardening {
```

Comment:
Could you add a comment of what needs to happen or must not happen in the comment?

The manual checks were conveying this in a more direct way (but broken in its own way :P).